.NET/Microsoft.Recognizers.Definitions.Common/Dutch/DateTimeDefinitions.cs

```diff
@@ -651,27 +651,20 @@ public static class DateTimeDefinitions
             { @"allsaintsday", new string[] { @"allerheiligen" } },
             { @"allsoulsday", new string[] { @"allerzielen" } },
             { @"christmaseve", new string[] { @"kerstavond" } },
-            { @"columbus", new string[] { @"columbusday" } },
+            { @"columbus", new string[] { @"columbusday", @"columbusday" } },

```

Comment:
There should be no duplicate values in the dictionary.

Reply:
fixed.